src/renderer/components/+catalog/catalog-add-button.tsx

```diff
@@ -64,14 +69,14 @@ export class CatalogAddButton extends React.Component<CatalogAddButtonProps> {
         open={this.isOpen}
         onOpen={this.onOpen}
         onClose={this.onClose}
-        icon={<Icon material="add" />}
```

Comment:
I think we need to set a `eslint` rule for this. Because they keep getting change back and forth. I would argue for keeping the space because we tend to do that on object literals.